src/openms/include/OpenMS/KERNEL/MSSpectrum.h

```diff
@@ -505,8 +541,13 @@ namespace OpenMS
     ConstIterator PosEnd(ConstIterator begin, CoordinateType mz, ConstIterator end) const;
 
     /// do the names of internal metadata arrays contain any hint of ion mobility data, e.g. 'Ion Mobility' 
+    /// (for spectra which represent an IM-frame)
     bool containsIMData() const;
 
+    /// get the Ion mobility data array (for spectra which represent an IM-frame)
+    /// @throws Exception::MissingInformation if IM data is not present
+    const MSSpectrum::FloatDataArray& getIMData() const;
```

Comment:
is the name of the data array set (e.g., to a CV term)? if so I would document it

Reply:
coming up in the next PR...